src/tag/Names.c

```diff
@@ -54,3 +54,15 @@ const char *const tag_item_names[TAG_NUM_OF_ITEM_TYPES] = {
 	[TAG_MUSICBRAINZ_RELEASETRACKID] = "MUSICBRAINZ_RELEASETRACKID",
 	[TAG_MUSICBRAINZ_WORKID] = "MUSICBRAINZ_WORKID",
 };
+
+const char *const cover_item_names[COVER_NUM_OF_ITEM_TYPES] = {
```

Comment:
This whole commit is obsolete, because MPD has cover art support now!

Reply:
Just for caryaudio.  Before MPD has cover art support, we develop it. And caryaudio just use this.